### ISSUE TITLE
Generic type alias of Tuple doesn't result in correct type

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#1350]
This sample reveals the type `tuple[int]`, but it should be `tuple[int, int, int]`.

```python
T = TypeVar("T")
Alias = Tuple[T, T, T]

def func(x: Alias[int]):
    reveal_type(x)
```

**Comment by erictraut** [comment]
Upon closer inspection, the revealed type is `Alias[int]` which is correct.
